src/Resources/config/storage/doctrine.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+use League\Bundle\OAuth2ServerBundle\Manager\AccessTokenManagerInterface;
+use League\Bundle\OAuth2ServerBundle\Manager\AuthorizationCodeManagerInterface;
+use League\Bundle\OAuth2ServerBundle\Manager\ClientManagerInterface;
+use League\Bundle\OAuth2ServerBundle\Manager\Doctrine\AccessTokenManager;
+use League\Bundle\OAuth2ServerBundle\Manager\Doctrine\AuthorizationCodeManager;
+use League\Bundle\OAuth2ServerBundle\Manager\Doctrine\ClientManager;
+use League\Bundle\OAuth2ServerBundle\Manager\Doctrine\RefreshTokenManager;
+use League\Bundle\OAuth2ServerBundle\Manager\InMemory\ScopeManager;
```

Comment:
Why isn't a `ScopeManager` in the Doctrine namespace? Maybe it's a typo/bug?

Reply:
Because it does not depend on Doctrine nor it is aware of it. 
I would rather challenge the fact that the service definition is located in the doctrine file, it's probably something needed in both in-memory and doctrine contexts.

Reply:
Yes, you're right. Maybe moving it to `services.php` would do the trick?

Reply:
I think so yes 👍